data/mainnet/pools.js

```diff
@@ -6310,4 +6310,36 @@ module.exports = [
       </div>
    `,
   },
+  {
+    chain: CHAINS_ID.ETH_MAINNET,
+    id: 'convex_cvxCRV',
+    type: POOL_TYPES.INCENTIVE,
+    contractAddress: addresses.V2.convex_cvxCRV.NewPool,
+    collateralAddress: addresses.V2.convex_cvxCRV.NewVault,
+    rewardAPY: [],
+    rewardTokens: [addresses.iFARM],
+    rewardTokenSymbols: ['iFARM'],
+    tradingApyFunction: {
+      type: TRADING_APY_TYPES.CONVEX,
+      params: ['cvxCRV'],
```

Comment:
I don't think this is correct. The vault is for single asset cvxCRV staking, not the cxvCRV-CRV Curve pool. So there won't be any `tradingApy`.